### PR TITLE
Copter: add gripper release FS_OPTION

### PR DESCRIPTION
**Pull request by IamPete1**
Adds a bit to the FS_OPTION param to allow the gripper to be released in the event of a failsafe.

### CODE REVIEW

File: ArduCopter/Parameters.cpp [hunk 1]
```diff
@@ -951,7 +951,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @DisplayName: Failsafe options bitmask
     // @Description: Bitmask of additional options for battery, radio, & GCS failsafes. 0 (default) disables all options.
     // @Values: 0:Disabled, 1:Continue if in Auto on RC failsafe only, 2:Continue if in Auto on GCS failsafe only, 3:Continue if in Auto on RC and/or GCS failsafe, 4:Continue if in Guided on RC failsafe only, 8:Continue if landing on any failsafe, 16:Continue if in pilot controlled modes on GCS failsafe, 19:Continue if in Auto on RC and/or GCS failsafe and continue if in pilot controlled modes on GCS failsafe
-    // @Bitmask: 0:Continue if in Auto on RC failsafe, 1:Continue if in Auto on GCS failsafe, 2:Continue if in Guided on RC failsafe, 3:Continue if landing on any failsafe, 4:Continue if in pilot controlled modes on GCS failsafe
+    // @Bitmask: 0:Continue if in Auto on RC failsafe, 1:Continue if in Auto on GCS failsafe, 2:Continue if in Guided on RC failsafe, 3:Continue if landing on any failsafe, 4:Continue if in pilot controlled modes on GCS failsafe, 5: Release Gripper
```
Comment (rmackay9) [inline ArduCopter/Parameters.cpp:954]: Sorry to be annoying but let's remove the little space between the "5:" and the "R".

**Comment by rmackay9** [comment]
This only applies to the RC failsafe but it could also apply to the GCS and battery failsafes.  Is it specifically only for the RC failsafe?  If so the only concern there is the param description for FS_OPTIONS doesn't make that clear.

Ideally I think we should handle this using scripts although we don't have the infrastructure in place to execute a failsafe action implemented as a script.

File: ArduCopter/Parameters.cpp (hunk 1 above)
Comment (IamPete1) [reply ArduCopter/Parameters.cpp]: fixed

**Comment by IamPete1** [comment]
@rmackay9 This applies to all fail-safes. 

I agree that it would be nice to do this in scripting, but you could argue that for the whole of gripper, this is a much  smaller change than the one that would be required to let scripts do fail safes.

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**